appSettings.json

```diff
@@ -15,10 +15,11 @@
       "sliceCount": "100000",
       "runIngestion": true,
       "runQuery": false,
-      "numQueries": 0,// specific num of queries, default 0 will run against entire query file
-      "ingestWithBulkExecution": false,
-      /* Number of parallel batches reading dataset file and queing for ingestion */
-      "numIngestionBatchCount": 1,
+      "numQueries": 0, // specific num of queries, default 0 will run against entire query file
+      "searchListSizeMultiplier": 5, // multiplier for search list, default 5
+      "MaxPhysicalPartitionCount": 100, // client max conccurrent connections
+      "BulkIngestion": false, // use bulk client for ingestion

```

Comment:
any reason to rename 'ingestWithBulkExecution' to 'BulkExecution'? If not revert it?